src/hev-config.c

```diff
@@ -13,8 +13,11 @@
 #include "hev-logger.h"
 #include "hev-config.h"
 
-static char srv_address[256];
-static char srv_port[8];
+static HevConfigServer srv = {
+    .addr = NULL,
+    .login = NULL,
+    .password = NULL
+};
```

Comment:
No need for explicit initialization.

Reply:
For this pointers is checking for NULL in code to determinate is login and password are sets.

Reply:
Yes. The default value of uninitialized global variables is 0. If you initialize even if the value is 0, more file space is required.